### PR TITLE
Optional shorturls, fixes #6

### LINKED ISSUE #6: Make URL shortening optional
There are privacy implications in using these third party services, and sometimes it's just not needed. Please implement an option to switch it off.

### PR DESCRIPTION
**Pull request by jcharaoui**
This should fix #6. Contrary to the current behavior, I set the option to False in the config sample file, as I believe it should not be the default to publish feed item URLs to a 3rd party service.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -186,7 +189,7 @@ def __fetch_feed(self, feed_info):
                             newsdate = "no date"
 
                     # Update the database. If it's a new issue, post it to the channel
-                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsurl, newsdate)
+                    is_new = self.__db.insert_news(feed_info[0], newstitle, newsitem.link, newsdate)
```
Comment (gehaxelt) [inline bot.py:192]: Why did you change this? Wouldn't this make the whole `newsurls` stuff in line 177 unnecessary, when the raw `newsitem.link` value is used? I guess this should stay `newsurl`?

Comment (jcharaoui) [inline bot.py:192]: The `newsurl` variable is still used when posting the link to IRC. By always storing the original URL in the database, toggling the short URLs option will avoid double posting previously seen feed items.

Comment (gehaxelt) [inline bot.py:192]: Ah, yeah, right. Guess it was late at night and I mixed things up. Looks good!
